.env.template

```diff
@@ -9,3 +9,4 @@ TRELLO_TOKEN=<your trello token>
 TRELLO_BOARD_ID=<your board id>
 TRELLO_TODO_LIST_ID = <the id of your todo column>
 TRELLO_DONE_LIST_ID = <the id of your done column>
+TRELLO_DOING_LIST_ID=<the id of your doing column>
```

Comment:
You have added the "Doing" list id here and in the test env file, but I don't see it being used anywhere. Is this actually necessary?